CairoFCCOS/FCCOSEth002.xml

```diff
@@ -0,0 +1,218 @@
+                            
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="44" to="81"/>
+                                <title type="complete" ref="LIT1146Argano#Tuesday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="81" to="120"/>
+                                <title type="complete" ref="LIT1146Argano#Wednesday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="121" to="159"/>
+                                <title type="complete" ref="LIT1146Argano#Thursday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.5">
+                                <locus from="160" to="192"/>
+                                <title type="complete" ref="LIT1146Argano#Friday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.6">
+                                <locus from="193" to="210"/>
+                                <title type="complete" ref="LIT1146Argano#Saturday"/>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.7">
+                                <locus from="210" to="226"/>
+                                <title type="complete" ref="LIT1146Argano#Sunday"/>
+                            </msItem>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="227" to="230"/>
+                            <title type="incomplete" ref="LIT2842RepCh125" cert="low"/>
+                            <textLang mainLang="gez"/> 
+                            <note>The text, incomplete at the beginning and end, is a malkǝʾ-hymn to Christ, apparently
+                                the <ref type="work" corresp="LIT2842RepCh125"/>. The catalogue description does not allow
+                            to ascertain whether this text is part of the original production, a later textual addition, or a material addition
+                            originating from another manuscript.</note>
+                        </msItem>
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="page" quantity="230">230</measure>
+                                    <dimensions type="outer" unit="cm">
```

Comment:
we once said it is desirable to use mm

Reply:
The reason I didn't convert them is that I suspect that whole numbers given in centimeters might be rounded up, and converting into mm would simulate more preciseness than might be present here. On the other hand, if the filtering and searching benefits for quantitative analyses would outweigh that, let's do it.